dev-scripts.sh

```
@@ -0,0 +1,353 @@
+    # Create global link
+    print_step "Creating global npm link..."
+    npm link
+    if [ $? -ne 0 ]; then
+        print_error "Failed to create npm link"
+        return 1
+    fi
+
+    # Link to backend
+    if [ -d "$tlbot_path/backend" ]; then
+        print_step "Linking to backend..."
+        cd "$tlbot_path/backend"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)